search_evaluation_framework/create-click-model.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+# Create a click model.
+docker compose run search_evaluation_framework java -jar /app/search-evaluation-framework.jar -o http://opensearch:9200 -c coec
```

Comment:
Running the command on event data collected by doing some queries and clicks in the user interface results in an error. 

Looks like the format of events isn't what the app is expecting although I did not dig deep enough as of now to say if the events are not how they should look like or the app is expecting the wrong input format.

[stacktrace.txt](https://github.com/user-attachments/files/18487112/stacktrace.txt)

Reply:
Looking at another PR (https://github.com/o19s/opensearch-search-quality-evaluation/pull/68) in opensearch-search-quality-evaluation it looks like Chorus, the UBI spec and the search quality eval framework are out of sync regarding how UBI events look like.

I think we should coordinate the necessary changes first and then see if we want to merge this PR before or after the changes.

Reply:
I got the field added that was the cause of the stacktrace.

Reply:
Now complaining about the unrecognized field `query_response_id` that I also had to add to make commands successful when looking at the opensearch-search-quality-evaluation PR (https://github.com/o19s/opensearch-search-quality-evaluation/pull/68).

So I think it makes sense to address this one once we're through the other PR.